resources/assets/js/designer/components/modalList/modal-forms-list.vue

```diff
@@ -23,6 +23,14 @@
                 </vuetable>
                 <pagination single="Form" plural="Forms" :perPageSelectEnabled="true" @changePerPage="changePerPage"
                             @vuetable-pagination:change-page="onPageChange" ref="pagination"></pagination>
+                <template slot="modal-footer">
```

Comment:
All of these modals need `<template slot="modal-footer">` changed to `<div slot="modal-footer">` in addition the closing `<div>` tag should be updated to `</div>`